glscopeclient/OscilloscopeWindow.cpp

```diff
@@ -391,6 +396,28 @@ bool OscilloscopeWindow::OnTimer(int /*timer*/)
 	return true;
 }
 
+void OscilloscopeWindow::OnPreferences()
+{
+    if(m_preferenceDialog)
```

Comment:
Is m_preferenceDialog ever initialized anywhere? It should be hidden, deleted, and set to null in ClearSession() so it doesn't persist across sessions, and initialized to null in the constructor.

Reply:
Just saw the nullptr init in the header. My comment about clearing in ClearSession() stands, though.